lib/CPAN/Meta.pm

```diff
@@ -398,7 +398,7 @@ sub save {
   my ($self, $file, $options) = @_;
 
   my $version = $options->{version} || '2';
-  my $layer = $] ge '5.008001' ? ':utf8' : '';
+  my $layer = "$]" >= 5.008001 ? ':utf8' : '';
```

Comment:
We can just always use `:utf8 and eliminate this version comparison here as well: the minimum version version has been 5.8.1 since 2.150010 (August 2016)